src/NUnitConsole/nunit-console/ResultReporter.cs

```diff
@@ -210,13 +210,11 @@ private void WriteNotRunResults(XmlNode result)
 
         #region Helper Methods
 
-        private static readonly char[] EOL_CHARS = new char[] { '\r', '\n' };
+        private static readonly char[] EOL_CHARS = { '\r', '\n' };
```

Comment:
Will it work in all compilers? I recall having difficulty with it. Pretty sure it won't build under CF 3.5.